frontend/src/app/header/Header.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import { AccountCircle, Mail } from '@material-ui/icons';
+import { 
+  AppBar,
+  Toolbar,
+  Typography,
+  IconButton,
+  Menu,
+  MenuItem,
+  Button,
+  Badge
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    title: {
+      flexGrow: 1,
+    },
+  }),
+);
+
+const Header = () => {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const auth = true;
+
+  const handleMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div className={classes.root}>
+      <AppBar position="static">
+        <Toolbar>
+          <Typography variant="h6" className={classes.title}>
+            tfti
+          </Typography>
+          <div>
+            <Button color="inherit">
+              Browse
+            </Button>
+            <Button color="inherit">
+              Create Event
+            </Button>
+            <Button color="inherit">
+              My Events
+            </Button>

```

Comment:
Were we going to add a friends tab? I know it's not in our main functionality list, but maybe we could just link to it and show an error: not implemented?

Reply:
Yeah that would be easy to add. I wonder if they are even going to get the code to play around with it or if it'll only be what we show them